pyaedt/modules/Boundary.py

```diff
@@ -354,6 +368,334 @@ class NativeComponentPCB(NativeComponentObject, object):
 
     def __init__(self, app, component_type, component_name, props):
         NativeComponentObject.__init__(self, app, component_type, component_name, props)
+        self._filter_map2name = {"Cap": "Capacitors", "Ind": "Inductors", "Res": "Resistors"}
+
+    @property
+    @pyaedt_function_handler()
+    def footprint_filter(self):
+        """Minimum component footprint for filtering."""
+        if self._app.settings.aedt_version < "2024.2":
+            return None
+        return self.filters.get("FootPrint", {}).get("Value", None)
+
+    @footprint_filter.setter
+    @pyaedt_function_handler()
+    @disable_auto_update
+    def footprint_filter(self, minimum_footprint):
+        """Set minimum component footprint for filtering.
+
+        Parameters
+        ----------
+        minimum_footprint : str
+            Value with unit of the minimum component footprint for filtering.
+        """
+        self.auto_update = False
```

Comment:
```suggestion
```
Is there a reason to keep this `auto_update = False` since you use the decorator to temporarily disable auto update ? If you want this method to ensure that the auto_update is set to False afterward then fine, but I don't think that's the case as it's the only method that gets this call.

Reply:
Yes, it is a residual from before I implemented the wrapper :)